taxi_service/settings.py

```diff
@@ -143,3 +144,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice in the file (lines 80 and 148). This redundancy should be removed to avoid confusion. Keep only one definition.